PixivDBManager.py

```diff
@@ -18,14 +18,14 @@ class PixivDBManager:
     """Pixiv Database Manager"""
     __config__ = None
 
-    def __init__(self, target='', config=None):
+    def __init__(self, target='', config=None,timeout=5*3600):
```

Comment:
5 minutes = 5 * 60, right?

the timeout is in second, right? meaning it is 5 hours?

Reply:
Ahm you're right, my mistake, 5*60 would be far more sane.